build.sbt

```diff
@@ -561,6 +563,23 @@ lazy val scioRepl: Project = Project(
   scioExtra
 )
 
+lazy val scioJmh: Project = Project(
+  "scio-jmh",
```

Comment:
Don't like the naming of this project, a better one might be `scio-bench[mark]` as it's more important that it's benchmarks not what it uses for benchmarks. Also #830